oslabs-beta/Kubiquity
# Add BarChart template for displaying memory useage

WHAT DOES THIS PR DO? 
1. Adds the react-bar-chart package to our project. 
2. Adds a MemoryBarChart component. 
3. Displays fake memory usage in the MemoryBarChart. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 6 && npm install && npm run dev`.
2. In a second terminal, run `npm start`. 
3. Verify that you can see the bar chart, and that the data on the bar chart reflects the fake pod data. 

ADDITIONAL NOTES: 
1. Rob built a cool bar chart using D3, but it is super difficult to figure out how to add on labels and such, so we're currently just using this crappy package. A possible stretch feature would be to upgrade to a better package, possibly D3. 
2. There are a lot of places to beef up the barChart, but they all should be noted within the code in comments. 
3. I noticed that our build script isn't working—going to put up a PR later today to address that.
